stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -447,15 +476,9 @@ public void onFocusChange(View v, boolean hasFocus) {
             public void onFocusChange(View v, boolean hasFocus) {
                 if (hasFocus) {
                     scrollRight();
-                }
-            }
-        });
-
-        mCvcNumberEditText.setOnFocusChangeListener(new OnFocusChangeListener() {
```

Comment:
Note: this was actually a (not-terribly visible) bug that I caught with testing for this feature! This call to `setOnFocusChangeListener` is completely overwritten by the call that happens just a few lines later!